integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonInsertIntoCommand.scala

```diff
@@ -276,7 +280,15 @@ case class CarbonInsertIntoCommand(databaseNameOp: Option[String],
         }
         throw ex
     }
-    Seq.empty
+    if(loadResultForReturn!=null && loadResultForReturn.getLoadName!=null) {
+      Seq(Row(loadResultForReturn.getLoadName))
+    } else {
+      rowsForReturn
```

Comment:
why are you returning number of rows instead of segment id here ? when will the code enter here  when load is success ?
can you add some comment ?

Reply:
partition table case, what will be rowsForReturn?

Reply:
It's not number of rows.
It's segment id  from partition table case.
Ok, Comment has been added.